### PR TITLE
Check some packages version for SLES15SP3

### PR DESCRIPTION
**Pull request by chcao**
This PR checks the version of package postgresql, mariadb and
python3, to see whether they are the expected version on SLES15SP3.

- Related ticket: https://progress.opensuse.org/issues/77797
- Verification run: 
http://openqa.suse.de/tests/5270810#step/check_package_version/1
http://openqa.suse.de/tests/5270812#step/check_package_version/1
http://openqa.suse.de/tests/5270813#step/check_package_version/1
http://openqa.suse.de/tests/5270814#step/check_package_version/1

### CODE REVIEW

**Review by tinawang123 — APPROVED** [review]
LGTM

File: tests/console/check_package_version.pm [hunk 1]
```diff
@@ -61,18 +40,18 @@ sub cmp_version {
...
-    my ($pcks, $pckv, $jsc) = @_;
+    my ($pcks, $pckv) = @_;
     record_info($pcks, "$pcks version check after migration");
     my $output = script_output("zypper se -xs $pcks | grep -w $pcks | head -1 | awk -F '|' '{print \$4}'", 100, proceed_on_failure => 1);
     my $out    = '';
     for my $line (split(/\r?\n/, $output)) {
         if (trim($line) =~ m/^\d+\.\d+(\.\d+)?/) {
             $out = $line;
-            record_soft_failure("$jsc, The $pcks version is $out, but request is $pckv") if (!cmp_version($pckv, $out));
+            record_soft_failure("The $pcks version is $out, but request is $pckv") if (!cmp_version($pckv, $out));
```
Comment (lemon-suse) [inline tests/console/check_package_version.pm:50]: I think you need update the com_packages to compare little version which is not stable as '0'.

It means you can skip the compare for little version if the expected version is 13.0.0. A quick thinking, change the '13.0.0' to '13' for example and compare corresponding bits is ok.

Comment (chcao) [reply tests/console/check_package_version.pm:50]: I don't think it is necessary to skip the little version, it meet the checking requirement.

**Review by lemon-suse — APPROVED** [review]

**Comment by lemon-suse** [comment]
LGTM. Anyway, I still have concern for the purpose of this version compare.

**Pull request merged**